### DISCUSSION TITLE
How can I install on Android Firefox?

### DISCUSSION THREAD

**Discussion by mrtootoo** [Cookie-AutoDelete/Cookie-AutoDelete#980] [Q&A / Support] [answered]
Mozilla disabled Cookie AutoDelete for FireFox Android a while back.  It does not seem it will ever be permitted again without using a nightly build and the creation of a Mozilla account.  Is there any way to side load or force an install of CAD to Android Firefox with out creating a Mozilla account?

**Comment by kennethtran93** [comment]
The only way I can think of at this time is to find an older version of Firefox Android (v68.11 appears to be the last version before the next major upgrade) that uses the old UI and whatnot.  Be aware that there may be security vulnerablities that may have been fixed in the newer versions and that you will be missing out on new features from the newer versions as well (68+). Also when manually installing this version, ensure that it does not auto-update through Play Store or equivalent app store.  This post may be of help: https://support.mozilla.org/en-US/questions/1300966

If you like the new UI or added new features, then you'll need the Nightly version with a Firefox Account or somebody's collection link that includes CAD webextension.

**Reply by mrtootoo** [reply]
Thanks.  I am currently using v68.  I tried the nightly version but did not want to create a Mozilla account.  I have found a couple of collection links, however, I could not figure out how to use them without a Mozilla account.

**Reply by kennethtran93** [reply]
> Thanks. I am currently using v68. I tried the nightly version but did not want to create a Mozilla account. I have found a couple of collection links, however, I could not figure out how to use them without a Mozilla account.

For the Nightly version, this blog post https://blog.mozilla.org/addons/2020/09/29/expanded-extension-support-in-firefox-for-android-nightly/ has steps for you to get custom collection going.  Note that only one collection can be used at a time.  Any addon/webextension not part of that collection may be disabled automatically.  This may include when you swap custom collections.

For the previous android version 68, you should be able to install it either by our extension page https://addons.mozilla.org/en-US/firefox/addon/cookie-autodelete/, or through searching it on the addons page.  Note that when searching (use 'cookie' term for easier match), you may need to broaden the filter applied if you are not seeing any results.

**Comment by mrtootoo** [comment, marked as answer]
> this blog post https://blog.mozilla.org/addons/2020/09/29/expanded-extension-support-in-firefox-for-android-nightly/ has steps for you to get custom collection going

 I was missing these instructions.  Thanks again!
